source/reference/minio-mc-admin/mc-admin-replicate.rst

```diff
@@ -82,7 +82,7 @@ Syntax
    Create or expand a site replication configuration.
    The configuration uses asynchronous site replication by default, as MinIO recommends.
 
-   If your circumstances require synchronous site replication, complete the configuration, then use :mc-cmd:`mc admin replicate update --sync`.
+   If your circumstances require synchronous site replication, complete the configuration, then use :mc-cmd:`mc admin replicate update --mode` with the ``sync`` option.
```

Comment:
OK I know this was already here but still, this feels like a really long way of saying

```suggestion
To enable synchronous site replication, create the replication using this command *first*.
Then use :mc-cmd:`mc admin replicate update --mode sync <mc admin replicate update --mode>` to update the configuration.
```

Reply:
:+1: I like it.